draft-kwbdgrr-tsvwg-net-collab-rqmts.md

```diff
@@ -152,49 +164,69 @@ Streaming video contains the occasional key frame ("I-frame") containing a full
 These are necessary to rebuild receiver state after loss of delta frames.
 The key frames are therefore more critical to deliver to the receiver than delta frames.
 
-Streaming video also contains audio frames which can be encoded separately and thus can be signaled separately.
-Audio is more critical than video for almost all applications, but its importance (relative to other packets in the flow) is still an application decision.
+Streaming video also contains audio frames which can be encoded
+separately and thus can be signaled separately (requirement:
+REQ-MEDIA-KEYFRAME).
+
+Audio is more critical than video for almost all applications, but its
```

Comment:
I would reword this as this is not inclusive. Video or signals are important for persons with inabilities. Refer to Total Conversation concept.

Reply:
new wording is 'many applications', and many (most) people can hear.  We are well above 80/20 threshold of the common case that audio is more critical than video, most of the time.  The ability of the receiver to change the priority by communicating to the ISP -- without cooperation of the sender -- is a requirement and gives a hearing-impaired user the ability to adjust video above audio.

Reply:
Agreed, especially the last part of your reply.